ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Genesis/Governor.hs

```diff
@@ -46,16 +61,16 @@ sharedCandidatePrefix ::
   SecurityParam ->
   AnchoredFragment (Header blk) ->
   Map peer (AnchoredFragment (Header blk)) ->
-  AnchoredFragment (Header blk)
+  (AnchoredFragment (Header blk), Map peer (AnchoredFragment (Header blk)))
 sharedCandidatePrefix (SecurityParam k) curChain candidates =
-  trunc
+  (trunc, suffixes)
```

Comment:
Document what `suffixes` is. Also, `trunc` is not just a prefix of `curChain` as the documentation claims. Depending on the value of `excess` some elements might be skipped.

Reply:
Maybe the documentation of `sharedCandidatePrefix` could say:
```
-- | Compute the fragment loeFrag between the inferred immutable tip and the
-- earliest intersection between @curChain@ and any of the @candidates@.
--
-- The inferred tip might be the anchor of @curChain@, if there are less than
-- k headers before the first intersection, otherwise it will be the kth ancestor
-- of the first intersection.
--
-- The function also yields the suffixes of the intersection of loeFrag with
-- every candidate fragment.
```

Reply:
Edited in e455562.